Mobile-Apps.md

```diff
@@ -1,15 +1,15 @@
-Mobile Apps - 3 Types to Consider (Hybrid, Cross Platform, Native)
-Hybrid:
+#Mobile Apps - 3 Types to Consider (Hybrid, Cross Platform, Native)
+#Hybrid:
   Ex: PhoneGap - Free open source software to bundle code for appstore.
   Advantages of Hybrid Apps: Can leverage existing web content and resources, single codebase across devices, lowest cost option
   Disadvantages of Hybrid Apps: Limited functionality to take advantage of advanced smartphone hardware features
   Timeframe/Cost: Simple App 3-6 months, $10K
-Cross Platform:
+#Cross Platform:
   Framework Examples: Appcelerator (java), Xamarin (.net, c#)
   Advantages of Cross Platform Apps: Single code base for all platforms, most cost effective for multiple platforms, deep hardware integration, compiles to native, fairly quick to market, great for corporate type applications.
   Disadvantages of Cross Platform Apps: Might require licensing fees, less flexibility with UI/UX as compared to native, lag time for native hardware features, not good for games.
   Timeframe/Cost: Simple App 6 months, $10-$50k
-Native:
+#Native:
```

Comment:
@bwromanski Would be great if you can take a use case of a new app that needs to be developed and then list some of the considerations that you would take into account in order to decide between Hybrid / Cross Platform / Native solutions.